bindings/csharp/KeystoneNET/KeystoneNET.Tests/Compilation.feature

```diff
@@ -1,20 +0,0 @@
-﻿Feature: Compilation
```

Comment:
The testing framework you used was really nice, but unfortunately is [not yet compatible with .NET Core / Standard](http://specflow.org/documentation/.NET-Support/).

Reply:
I'm not very expert about code/standard and the like but on their website I read only about ".net core".

Specflow is composed of an extension of VS that only serve to generate the *.feature's code behind (this should not pose any compatibility issues) and internally it uses NUnit (and also this should not pose any compatibility issues). Also, I guess you can target .NET "something" in your test app and .net standard in everything else? Anyhow, I should try.

Reply:
.NET Core replaces the Framework and runs everywhere instead of just Windows; .NET Standard replaces Portable Class Libraries. Thus SpecFlow currently only targets Windows, and requires VS (except Express) or msbuild. We could use .NET Framework for the tests since it supports .NET Standard, but this would mean that tests are only available on Windows.

Finally, the generated code uses NUnit **and** SpecFlow (see [here](https://github.com/keystone-engine/keystone/pull/344/files/3eba118b711b807774d7be3a59b992ce02581387#diff-3e0003d485808a442aceea5f2196d7ceL5)).

Reply:
Ok then, altough I'm still pretty sure you can have the "test" assembly compiled with a given version of the framework, and hence supporting specflow, and all the rest compiled in .net standard.
But that's fine anyhow, as long as there are unit tests.